schemas/com.hipchat.sauna.commands/send_room_notification/jsonschema/1-0-0

```diff
@@ -0,0 +1,34 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for a Hipchat room notification command",
+	"self": {
+		"vendor": "com.hipchat",
+		"name": "send_room_notification",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"roomIdOrName": {
+			"type": "string",
+			"minLength": 1,
+			"maxLength": 100
+		},
+		"color": {
+			"enum": ["yellow", "green", "red", "purple", "gray", "random"]
+		},
+		"message": {
+		    "type": "string",
+			"minLength": 1,
+			"maxLength": 10000
+		},
+		"notify": {
+			"type": "boolean"
+		},
+		"messageFormat": {
+			"enum": ["html", "text"]
+		}
+	},
+	"required": ["roomIdOrName", "message"],
```

Comment:
It's not required and set to `false` by default as per https://www.hipchat.com/docs/apiv2/method/send_room_notification. `null` probably shouldn't match.